jep/1/README.adoc

```diff
@@ -316,15 +298,15 @@ the idea is applicable to the entire community and not just the sponsor. Just
 because an idea sounds good to the sponsor does not mean it will work for most
 people in most areas where Jenkins is used.
 
+A <<JEP Reviewer>> should be selected before the JEP is submitted,
+but may also be added or changed any time before a document is reviewed.
```

Comment:
I see no specific need in this requirement, but OK.
I would just say that "JEP Reviewers should be selected via a discussion in the mailing list or in the Governance meeting, discussion in the pull request is NOT enough"